cmd/csaf_provider/config.go

```diff
@@ -23,10 +23,11 @@ import (
 const (
 	// The environment name, that contains the path to the config file.
 	configEnv                = "CSAF_CONFIG"
-	configPrefix             = "/usr/lib/csaf"
+	configKeyPrefix          = "/usr/lib/csaf"
```

Comment:
Is there a reason to have the keys still in `/usr/lib/csaf/`? What does the FHS suggest?

Reply:
Without checking I also believe that the openpgp key material is configuration to the provider and therefore should also be under `/etc`. Thanks for the pointer.